src/datachain/cli.py

```diff
@@ -400,6 +400,18 @@ def get_parser() -> ArgumentParser:  # noqa: PLR0915
         "--edatachain-file",
         help="Use a different filename for the resulting .edatachain file",
     )
+    parse_pull.add_argument(
```

Comment:
[C] I still don't know if it is better to expose these or uuid to the user

Reply:
I think we should let client work with name / version. `uuid` is not really practical, and also user cannot really set it.